src/gtk/listbox.c

```diff
@@ -133,7 +133,7 @@ listbox_add_columns (gftp_window_data *wdata)
    /* icon */
    //renderer = gtk_cell_renderer_pixbuf_new ();
    renderer = g_object_new (GTK_TYPE_CELL_RENDERER_PIXBUF,
-                           "xalign", 0.5,     /* justify left */
+                           "xalign", 0.5,     /* justify center */
```

Comment:
I almost hate to ask this but maybe this should be in a separate PR. If it's too much of a pain, then I'll take this as is in this PR once those two other minor changes above are addressed. Thanks for your contributions!